### DISCUSSION TITLE
can you fix keybinds for nintendo games?

### DISCUSSION THREAD

**Discussion by LikesJazz** [starg3n/starg3n.github.io#5] [Help Centre] [answered]
Lifesaver for playing in class but the keybinds for Nintendo games are really messed up, thanks!

**Comment by starg3n** [comment, marked as answer]
Hi Jazz.

I assume "Nintendo games" means the GBA collection, but please correct me if you mean the DS emulator :)
I tested the controls on [Super Mario Advance](https://starg3n.github.io/gba/launcher.html#supermarioadvance) and [Tetris Worlds](https://starg3n.github.io/gba/launcher.html#tetris_worlds). The controls seem to work fine for me, Z and X work for the buttons, (not sure which button works for which, haha) and the arrow keys work for the dpad. not sure what problem you had, but it didn't replicate for me :(

sorry for the late response!
I'm glad that it's a lifesaver in class, feel free to give it a star if you like it :)

**Reply by LikesJazz** [reply]
Howdy! 

Apologies for the confusion, but I meant to ask if there could be a possibility to have an option where you can use more common keybinds like WASD, Q, and E instead of having to use arrow keys and Z, X. 

Thanks!

**Reply by starg3n** [reply]
Hi again!

Those are... very interesting keybinds.
Personally, I've never seen anyone use these types of controls, and I'm not so sure two sets of controls would be possible.
I'll try this though, if I can get it working I'll push it as an update :)
